### ISSUE TITLE
List existing reports that the user has access to (through their organisation permissions) - API endpoint

### ISSUE THREAD

**Issue by pbastia** [bcgov/cas-reporting#125]
#### Description:

This is once we have a `report` table, to allow a user to see the entries linked to their own user or organisations.

#### Acceptance Criteria:

Given I am a reporting app user
When I have report entries in the database that I have the right to access
Then I can see them on the screen (crudely)

#### Development Checklist:

- [ ] a next page (maybe `/reporting/reports` ?)
- [ ] a crude collection of `<li>` elements
- [ ] a `/reports` API endpoint
- [ ] mock data in the database is enough
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By

This ticket is blocked by #124 

#### Notes:

-

**Comment by mikevespi** [comment]
What are we linking to for the first year? Are we going to have links to SWRS and CIIP?

**Comment by pbastia** [comment]
Long term - yes, is my guess? But this wasn't the scope I had in mind for this card. This is just "I can see my own reports"

**Comment by patriciarussellCAS** [comment]
This may also relate to ticket #145 - how they see historical data, who sees it, and how can it be used comparatively with current data entered, if at all?

**Comment by patriciarussellCAS** [comment]
To confirm: Is this a duplicate? Can we close?
